README.md

```diff
@@ -127,6 +127,36 @@ checkerFramework {
 }
 ```
 
+### Multi-project builds
+
+By default, checkers are run on all subprojects of the project to which the plugin
+is applied. If you wish to disable this behavior, set the `applyToSubprojects`
```

Comment:
Why would one wish to disable the behavior?  Can you mention a scenario or how a user can tell whether he/she is in that category?

Reply:
I have added an example.